src/main/java/com/checkout/customers/CustomersClient.java

```diff
@@ -0,0 +1,16 @@
+package com.checkout.customers;
+
+import com.checkout.common.IdResponse;
+
+import java.util.concurrent.CompletableFuture;
+
+public interface CustomersClient {
+
+    CompletableFuture<CustomerDetailsResponse> get(String customerId);
+
+    CompletableFuture<IdResponse> create(CustomerRequest customerRequest);
+
+    CompletableFuture<Void> update(String customerId, CustomerRequest customerRequest);
```

Comment:
Please double check if the response is `Void`.

Reply:
Yes, the response is void.